### PR TITLE
feat: team challenge info added to challenge header

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- team challenge info added to challenge header

Screenshot 2024-03-14 at 9 01 14
Screenshot 2024-03-14 at 9 01 28

### CODE REVIEW

File: src/graphql/query/teamsGoals.graphql [hunk 1]
```diff
@@ -3,8 +3,16 @@ query GetGoals($teamId: Int, $limit: Int) {
 		totalCount
```
Comment (dyersituations) [inline src/graphql/query/teamsGoals.graphql:3]: Update this to use the new `goals` naming to get rid of this deprecation warning in GitHub.

File: src/graphql/query/teamsGoals.graphql [hunk 2]
```diff
@@ -3,8 +3,16 @@ query GetGoals($teamId: Int, $limit: Int) {
...
 		values {
 			id
-			description
 			name
+			description
+			descriptionAuthor {
+				id
+				name
+				image {
+				id
```
Comment (dyersituations) [inline src/graphql/query/teamsGoals.graphql:12]: Indent these two fields.

File: src/components/Lend/LoanSearch/ChallengeHeader.vue [hunk 3]
```diff
@@ -1,17 +1,103 @@
...
 	name: 'ChallengeHeader',
+	components: {
+		KvUserAvatar,
+	},
 	props: {
 		challengeData: {
 			type: Object,
 			default: () => ({}),
 		},
+		teamData: {
```
Comment (dyersituations) [inline src/components/Lend/LoanSearch/ChallengeHeader.vue:67]: Does a story need to be updated with this new data?

File: src/pages/Lend/LoanSearchPage.vue [hunk 4]
```diff
@@ -179,7 +187,7 @@ export default {
 			return this.enableChallengeHeader && !!this.challengeData?.id;
 		},
 	},
-	created() {
+	async created() {
```
Comment (dyersituations) [inline src/pages/Lend/LoanSearchPage.vue:190]: Is `async` needed?

File: src/components/Lend/LoanSearch/ChallengeHeader.vue (hunk 3 above)
Comment (roger-in-kiva) [reply src/components/Lend/LoanSearch/ChallengeHeader.vue:67]: We don't have a story for this.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**